File: backend/doctors/models/doctorSpeciality.py
from django.db import models


class DoctorSpeciality(models.Model):
    """Doctor specialities"""

    name = models.CharField(max_length=30, unique=True)
    active = models.BooleanField(default=True)

    def __str__(self) -> str:
        return str(self.name)

File: frontend/src/types/doctors/DoctorSpeciality.ts
import { ID } from "../utils/id";

export type DoctorSpeciality = {
    id?: ID;
    name: string;
    active: boolean;
};

export const newDoctorSpeciality = (): DoctorSpeciality => {
    return {
        name: "",
        active: true,
    };
};
